OpenRA.Mods.Common/UtilityCommands/UpgradeRules.cs

```diff
@@ -2249,6 +2249,14 @@ internal static void UpgradeWeaponRules(int engineVersion, ref List<MiniYamlNode
 					}
 				}
 
+				if (engineVersion < 20150828)
+				{
+					if (depth == 2 && parentKey == "Projectile" && !node.Key.StartsWith("Sequences") && node.Key.StartsWith("Sequence"))
```

Comment:
This does not check for `Bullet` as a value so it will break `Missile` and all the other projectiles which still use `Sequence`.